Added Social Media Links as well as APIs.

nzec opened the pull request:
I have also added APIs for Flickr and Twitter linked to FOSSASIA's accounts.

nzec committed:
OKAY

nzec committed:
asd

nzec committed:
asd

nzec committed:
sad

nzec committed:
sdasdsadasd

nzec committed:
MY GAWD!

abishekvashok commented on CNAME:
```diff
@@ -1 +0,0 @@
-gci17.fossasia.org
```
This is a required file as this site is hosted on github (gh-pages)

nzec replied:
Oh okay. I'll get that back!

nzec replied:
Done

abishekvashok reviewed (changes requested):
Please try with the current font included in the theme, a suggestion only though

nzec committed:
CNAME

duttaditya18 commented:
And what should I do with the font?

abishekvashok commented:
@duttaditya18 sticking with the current font would be good, i guess.

duttaditya18 commented:
I don't think I changed any font.

abishekvashok commented on css/font-awesome.css:
```diff
@@ -0,0 +1,2337 @@
+/*!
```
I was mentioning this file

nzec replied:
That is for symbols. (FontAwesome delivers logos and symbols as fonts) FontAweosme which was earlier linked was outdated, so I linked the newer one. Nothing special changed other than the Google+ logo.

duttaditya18 commented:
@abishekvashok Can this pull request be merged now?

duttaditya18 commented:
@abishekvashok Also, who will accept my contribution on Code-In? How am I supposed to do other tasks if this one does not get approved?

poush reviewed (changes requested):
Update only current minified font awesome CSS file. Do not add a new one.

abishekvashok commented:
> @abishekvashok Also, who will accept my contribution on Code-In? How am I supposed to do other tasks if this one does not get approved? Sorry for bothering you too much!

Some mentor would merge your code, after you submit a task for review if you have met all conditions the task will be accepted and then you can go for other tasks. :) You will also get peer reviews just like I did.

akarshan96 commented:
Hi, @duttaditya18 can you please squash the commits and give relevant commit message to the single squashed commit. Rest looks fine 👍

nzec committed:
Font Awesome Fix

nzec committed:
Added Social Links as well as APIs.

Added Twitter and Flickr API.
Added Social Icons and Correct Links.
Updated Font Awesome.

nzec committed:
Merge remote-tracking branch 'origin/gh-pages' into gh-pages

duttaditya18 commented:
@poush Updated the minified FontAwesome CSS file only and the fonts.
@akarshan96 Squashed the commits into 94c9ac9 (Added Social Links as well as APIs.)

nzec committed:
Merge remote-tracking branch 'upstream/gh-pages' into gh-pages

nzec committed:
Formatting.

duttaditya18 commented:
@poush @akarshan96 @abishekvashok  Just tell me what should be done next. I have done everything that was asked.

CosmicCoder96 commented:
@duttaditya18 You have not squashed the commits at all :) as you can see your PR contains 12 commits.

duttaditya18 commented:
@CosmicCoder96 How do I squash it using GitKraken?
